app/models/katello/subscription_status.rb

```diff
@@ -58,6 +64,8 @@ def to_status(options = {})
         INVALID
       when 'unsubscribed_hypervisor'
         UNSUBSCRIBED_HYPERVISOR
+      when 'simple_content_access'
```

Comment:
I don't think this is needed based on my suggestions

Reply:
Removed